go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/filecoin-project/go-address v0.0.2-0.20200218010043-eb9bb40ed5be
 	github.com/filecoin-project/go-amt-ipld/v2 v2.1.0
-	github.com/filecoin-project/go-bitfield v0.1.0
+	github.com/filecoin-project/go-bitfield v0.1.2-0.20200727181427-9882314b1ea4
```

Comment:
Need to merge https://github.com/filecoin-project/go-bitfield/pull/43 and cut a release.